README.md

```diff
@@ -131,22 +127,23 @@ Exam objectives that outline of the knowledge, skills and abilities that a Certi
 
     - [Kubernetes Documentation > Concepts > Configuration > Secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
 
-- Use container runtime sandboxes in multi-tenant environments (e.g. gvisor, kata containers
+- Understand and implement isolation techniques (multi-tenancy, sandboxed containers,
+etc.)
 
     - [Kubernetes Documentation > Concepts > Security > Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/#what-about-sandboxed-pods)
     - [Kubernetes Documentation > Concepts > Containers > Runtime Class](https://kubernetes.io/docs/concepts/containers/runtime-class/)
     - [gvisor](https://gvisor.dev/docs/user_guide/quick_start/kubernetes/)
     - [kata containers](https://katacontainers.io/)
 
-- Implement pod to pod encryption by use of mTLS
+- Implement Pod-to-Pod encryption using Cilium
```

Comment:
Add a link to Cilium docs (mTLS should still be relevant as Cilium also implements it)